graphite.go

```diff
@@ -126,8 +127,8 @@ func (graphite *Graphite) SimpleSend(stat string, value string) error {
 
 // NewGraphiteHost is a factory method that's used to create a new Graphite
 // connection given a hostname and a port number
-func NewGraphite(host string, port int) (*Graphite, error) {
-	Graphite := &Graphite{Host: host, Port: port}
+func NewGraphite(host string, port int, protocol string) (*Graphite, error) {
```

Comment:
I'd rather these be new signatures instead of adding this requirement to the existing API. Avoid breaking API changes at all costs please :)